### PR TITLE
Upgrade PhpStorm EAP to 141.1717

### PR DESCRIPTION
**Pull request by bchatard**
Hello,

I try to follow guideline from #879. Let me know if some parts are missing or if there are some mistakes.

Hope this help

Replace PR #1074 

// cc @sebroeder

### CODE REVIEW

**Comment by deizel** [comment]
:+1: - Can we get this merged before bumping the version again? (141.1815 was released earlier today.)

Since this Cask tracks PhpStorm 9.x, this PR helps target the correct `WebIde90` directories (vs `WebIde80`).

```console
$ find ~ -type d -name '*WebIde90'
/Users/deizel/.WebIde90
/Users/deizel/Library/Application Support/WebIde90
/Users/deizel/Library/Caches/WebIde90
/Users/deizel/Library/Logs/WebIde90
/Users/deizel/Library/Preferences/WebIde90
```

File: Casks/phpstorm-eap.rb [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
+
+  caveats <<-EOS.undent
+    #{token} requires Java 6 like any other IntelliJ-based IDE.
+    You can install it with
+    brew cask install caskroom/homebrew-versions/java6
+    The vendor (JetBrains) doesn't support newer versions of Java (yet)
+    due to several critical issues, see details at
+    https://intellij-support.jetbrains.com/entries/27854363
+  EOS
+end
```
Comment (vitorgalvao) [inline Casks/phpstorm-eap.rb]: It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

**Comment by vitorgalvao** [comment]
@deizel I’d like for @sebroeder’s input first, since he has been working hard on this, and I do not wish to undo any of the work.

**Comment by sebroeder** [comment]
Thanks for your work @bchatard and sorry for my late reply @deizel, @vitorgalvao.

The cask looks good, the name stanza is missing, though and homepage URL could be https as well. Maybe you can change this while you add the missing newline at the end of the file, @bchatard? You can just add another commit to your existing branch and push it to GitHub, no need for a new Pull Request. If you want to, you could also bump the version to the newer 141.1815 (the Jetbrains folks are killing me with there insane release frequency).

Let me know if you need any more help (I am a little busy at the moment but I’ll try to answer as quickly as possible). The official [Cask Language Reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md) is a recommended read, too.

**Comment by bchatard** [comment]
@sebroeder & @vitorgalvao  thanks for your feedback. I just push the fixes.
I do not upgrade version to the last one in this PR. I'm agree with @deizel, this is a good step to ensure  cask quality before upgrading to the new version. 
When this PR will be ok, I (or someone else) will push a PR to upgrade version.

**Comment by bchatard** [comment]
I don't understand this error, could you help me please?

**Comment by vitorgalvao** [comment]
It’s just Travis chocking. Don’t worry.

**Comment by bchatard** [comment]
@vitorgalvao thanks!

**Comment by deizel** [comment]
> When this PR will be ok, I (or someone else) will push a PR to upgrade version.

Looks like there is a PR waiting already :) #1102

/ping @sebroeder

**Comment by deizel** [comment]
@vitorgalvao Seems the 9.0 EAP is now [closed](https://confluence.jetbrains.com/display/PhpStorm/PhpStorm+Early+Access+Program) - do the casks get deleted now? (see 4e5408bb9ba2d8723e41b7395dc2a8469f3d41b5)

@sebroeder, @vitorgalvao, the [stable cask](https://github.com/caskroom/homebrew-cask/blob/master/Casks/phpstorm.rb) could use some of the improvements from this PR?

**Comment by sebroeder** [comment]
@deizel The casks for EAPs do not get deleted, even if an EAP is temporarily closed. This makes it very easy for us to just update the cask when a new EAP starts. The PhpStorm EAP is already open again.

Feel free to bring the changes from #879 over to the stable cask. I just didn't do it yet because I want to finish #879 first and personally use only the bundled-jdk versions. [Things might get easier in the future, though](879#issuecomment-121670568).

**Comment by sebroeder** [comment]
> Looks like there is a PR waiting already :) #1102

Unfortunately this one is not based on your improvements.

**Comment by jawshooah** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 16f32b7. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by vitorgalvao** [comment]
Missed your request, @sebroeder, apologies. Thank you for the catch, @jawshooah.

**Pull request closed without merging**